src/main/java/com/adobe/commerce/cif/model/cart/ShippingMethod.java

```diff
@@ -29,7 +29,7 @@
     protected String description;
     
     @ApiModelProperty(value = "The price of the shipping method aka shipping cost.", required = true)
-    protected Price price;
```

Comment:
I recommend renaming the field name to `cost`, since shipping fee is a cost you pay, not a price. The comment already uses cost.